modules/luci-mod-admin-full/luasrc/model/cbi/admin_network/wifi.lua

```diff
@@ -818,21 +829,12 @@ if hwtype == "mac80211" or hwtype == "prism2" then
 	end
 	ieee80211r.rmempty = true
 
-	nasid = s:taboption("encryption", Value, "nasid", translate("NAS ID"),
-		translate("Used for two different purposes: RADIUS NAS ID and " ..
-			"802.11r R0KH-ID. Not needed with normal WPA(2)-PSK."))
-	nasid:depends({mode="ap", encryption="wpa"})
-	nasid:depends({mode="ap", encryption="wpa2"})
-	nasid:depends({mode="ap-wds", encryption="wpa"})
-	nasid:depends({mode="ap-wds", encryption="wpa2"})
-	nasid:depends({ieee80211r="1"})
-	nasid.rmempty = true
-
 	mobility_domain = s:taboption("encryption", Value, "mobility_domain",
-			translate("Mobility Domain"),
-			translate("4-character hexadecimal ID"))
+		translate("Mobility Domain"),
+		translate("4-character hexadecimal ID" ..
+			"<br />(Default generated from SSID)"))
 	mobility_domain:depends({ieee80211r="1"})
-	mobility_domain.placeholder = "4f57"
+	mobility_domain.placeholder = ut.exec("echo " .. ut.shellquote(wnet:get("ssid")) .. " | md5sum | head -c 4")
```

Comment:
Note that this is only a placeholder; is it the same algorithm used by OpenWrt?

Reply:
Yes
https://github.com/openwrt/openwrt/blob/master/package/network/services/hostapd/files/hostapd.sh#L427